### DISCUSSION TITLE
Interpretation of quoted absolute paths

### DISCUSSION THREAD

**Discussion by gitraphha** [RsyncProject/rsync#403] [General]
Dear all,
I just noted a behaviour change where I am unsure whether this is by design or a bug.
I am using backintime to do backups to a remote server over ssh. backintime generates an elaborated rsync command and runs it. Now my backups stopped working under ubuntu 22.10. I tracked down the problem to a change in the rsync version (22.04: 3.2.3; 22.10: 3.2.5). The commandline generated by backintime is exactly the same.
The command line contains sth like: user@buserver:"/absolute/path/spec" as the target (note that the path is double-quoted). Additionally ssh is used: i.e. "--rsh=ssh -o ..."

The final directory (e.g. spec) does not exist yet, so rsync tries to create it.

With rsync3.2.3 this is working just fine, whereas with 3.2.5, rsync tries to create a directory that looks like $HOME"/absolute/path/spec" (e.g. /home/user/"/absolute/path/spec").

This $HOME-path is exactly the (absolute) path the user ends up in when connecting via ssh.

Coming back to my question... is this a bug or a feature?

Thanks for any hint, Raph

**Comment by WayneD** [comment]
See: https://download.samba.org/pub/rsync/NEWS#3.2.4
Specifically the suggestion to add `export RSYNC_OLD_ARGS=1` to a script that does its own arg quoting.

**Reply by gitraphha** [reply]
Wonderful, thanks for your quick reply Wayne.

For all those who stumbled upon this thread because of troubles with backintime/rsync, here's a link to a long discussion, seems like it's a bit more complicated than just adding "--old-args" in the "Settings->Expert Options->Paste additional options to rsync" text field.
https://github.com/bit-team/backintime/issues/1247
Seems it will be fixed in the next release of backintime though.

Best, Raph
